client/my-sites/upgrades/domain-management/domain-connect/constants.js

```diff
@@ -0,0 +1,10 @@
+export const actionType = {
```

Comment:
Thank you! I was actually looking for something like Go's `iota`. This looks perfect!